modules/raft-api/src/main/java/org/apache/ignite/internal/raft/RaftGroupConfigurationSerializer.java

```diff
@@ -32,6 +32,17 @@ public class RaftGroupConfigurationSerializer extends VersionedSerializer<RaftGr
     /** Serializer instance. */
     public static final RaftGroupConfigurationSerializer INSTANCE = new RaftGroupConfigurationSerializer();
 
+    /** Unknown index value. */
+    private static final long UNKNOWN_INDEX = -1L;
+
+    /** Unknown term value. */
+    private static final long UNKNOWN_TERM = -1L;
```

Comment:
These seem to belong to `RaftGroupConfiguration` itself and be public as some code external to raft (and our additions to raft) will use them to handle 'no index+term information' situation.

Reply:
moved